src/script/sign.js

```diff
@@ -1,83 +1,57 @@
-const email_input = document.querySelector("#signin_email");
-const password_input = document.querySelector("#signin_password");
-const error_email = document.querySelector("#error_email");
-const error_password = document.querySelector("#error_password");
-const eyes = document.querySelector(".eye_button");
+import createError from "./error/createError.js";
+import clearError from "./error/clearError.js";
+
+const email = document.querySelector("#sign_email");
+const password = document.querySelector("#sign_password");
+const emailError = document.querySelector("#error_email");
+const passwordError = document.querySelector("#error_password");
+const eyeButton = document.querySelector(".eye_button");
+const emailRegex = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-za-z0-9\-]+/;
 
 // 이메일 검사
 function validateEmail() {
+    const isEmpty = email.value.trim() === "";
+
     // 이메일 값이 없는 경우
-    if (email_input.value.trim() === "") {
-        email_input.style.border = "1px solid var(--Linkbrary-red)";
-        error_email.innerText = "이메일을 입력해 주세요.";
-        error_email.style.display = "block";
+    if (isEmpty) {
+        createError(email, emailError, "이메일을 입력해 주세요.");
+        return;
     }
     // 올바른 이메일 주소가 아닐 경우
-    else if (!isValidEmail(email_input.value)) {
-        email_input.style.border = "1px solid var(--Linkbrary-red)";
-        error_email.innerText = "올바른 이메일 주소가 아닙니다.";
-        error_email.style.display = "block";
+    else if (!isValidEmail(email.value)) {
```

Comment:
상단에 return 문이 있기 때문에 else if문이 아닌 if문으로 작성하시면 됩니다.

```suggestion
    if (!isValidEmail(email.value)) {
```